src/components/ciPipeline/CiPipelineSourceConfig.tsx

```diff
@@ -134,8 +138,14 @@ export function CiPipelineSourceConfig({
                             <div>
                                 {!baseText && (
                                     <div className="flex left">
-                                        <div className="dc__ellipsis-right">{sourceValueBase}</div>
-                                        <Info className="icon-dim-12 fcn-5 ml-4" />
+                                        <div
+                                            className={`dc__ellipsis-right ${
+                                                sourceValue === GIT_BRANCH_NOT_CONFIGURED ? 'cr-5' : ''
+                                            }`}
+                                        >
+                                            {sourceValueBase}
+                                        </div>
+                                        {sourceValue !== '--' ? <Info className="icon-dim-12 fcn-5 ml-4" /> : ''}
```

Comment:
please change this to {sourceValue !== '--' && <Info className="icon-dim-12 fcn-5 ml-4" />}

Reply:
done